### PR TITLE
AT_071.01_Java | Entity > Arithmetic Function: Edit an existing record

### PR DESCRIPTION
**Pull request by Maariiaanaa**
https://trello.com/c/IeWn1aE0/390-at07101java-entity-arithmetic-function-edit-an-existing-record

### CODE REVIEW

File: src/test/java/EntityArithmeticFunctionEditExistingRecordTest.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        findElement(F1_FIELD).clear();
+        findElement(F1_FIELD).sendKeys("9");
+        findElement(F2_FIELD).clear();
+        findElement(F2_FIELD).sendKeys("5");
+        getWait().until(ExpectedConditions.attributeToBe(By.id("div"), "value", "1.8"));
+        getDriver().findElement(SAVE_BUTTON).click();
+    }
+
+    @Test
+    public void testEditExistingRecordTest() {
```
Comment (guram21) [inline src/test/java/EntityArithmeticFunctionEditExistingRecordTest.java:43]: Fix the name of the test method
```suggestion
    public void testEditExistingRecord() {
```

File: src/test/java/EntityArithmeticFunctionEditExistingRecordTest.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+        createNewRecord();
+        clickEdit();
+        editFields();
+
```
Comment (guram21) [inline src/test/java/EntityArithmeticFunctionEditExistingRecordTest.java:48]: I suggest validating how many records exist by adding Assertion

**Review by Jenny-ai — APPROVED** [review]
agree with adding assertion for number of rows, or even check that we edit the correct record

File: src/test/java/EntityArithmeticFunctionEditExistingRecordTest.java (hunk 1 above)
Comment (Maariiaanaa) [reply src/test/java/EntityArithmeticFunctionEditExistingRecordTest.java]: Fixed the name of the test method, thank you

File: src/test/java/EntityArithmeticFunctionEditExistingRecordTest.java (hunk 2 above)
Comment (Maariiaanaa) [reply src/test/java/EntityArithmeticFunctionEditExistingRecordTest.java]: Added assertion

**Comment by Maariiaanaa** [comment]
> agree with adding assertion for number of rows, or even check that we edit the correct record

Added assertions for number of rows and to check that we edit the correct record

**Review by guram21 — APPROVED** [review]

**Pull request merged**